knimin/lib/data_access.py

```diff
@@ -3460,7 +3460,7 @@ def get_sample_plate_list(self):
             return plates
 
     def extract_sample_plates(self, sample_plate_ids, email, robot, kit_lot,
-                              tool, notes=None):
+                              tool, notes=None, names=[]):
```

Comment:
it is dangerous to use a default like this, please change to `names=None`, test for it, and set the default within the method